packages/resin-event-log/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Convert to JS
```

Comment:
We planned to convert everything to ES6 or TS eventually.
These modules are trivial and plain JS should be enough.
It makes the repo simpler and doesn't require the build step.

But if we decide to use TS it's easier to convert from JS. And for the initial conversion I've just picked the files generated by CS and cleaned them up a bit.